bot/cogs/git.py

```diff
@@ -131,6 +131,24 @@ async def init(self, ctx):
             )
         )
         await ctx.send(msg)
+    @git.command(help'Prints info on the git pull command')
+    async def pull(self, ctx):
+        msg = ManMessage(
+            name='git-pull'
+            synopsis='git pull git pull [options] [<repository> [<refspec>...]]',
+            options=[
+                ('-q, --quiet'),('-v, --verbose'),
```

Comment:
One option is a tuple of (name, description). Please add a description for each option. Empty descriptions should also be avoided if possible.